src/bioregistry/align/ncbi.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+
+"""Align NCBI with the Bioregistry."""
+
+from typing import Any, Dict, List, Mapping
+
+from bioregistry.align.utils import Aligner
+from bioregistry.external.ncbi import get_ncbi
+
+__all__ = ['NcbiAligner']
+
+
+class NcbiAligner(Aligner):
+    """Aligner for NCBI xref registry."""
+
+    key = 'ncbi'
+    getter = get_ncbi
+    curation_header = ('name', 'homepage', 'example')
+
+    def prepare_external(
+        self,
+        external_id: str,
+        external_entry: Dict[str, str],
+    ) -> Mapping[str, Any]:
+        """Stub for preparing NCBI xref data to be added to the bioregistry.
+
+        The NCBI getter does all the necessary processing, so use the superclass'
+        default implementation.
+
+        :param external_id: the key of the entry in the incoming data
+        :param external_entry: the actual entry data from the incoming data
+
+        :returns: the processed entry data
+        """
+        return super().prepare_external(external_id, external_entry)
```

Comment:
why bother overriding this? just don't provide any implementation at all

Reply:
Because it's an abstractmethod, so I have to. I guess I could remove that annotation too if that makes sense.

Reply:
Yup. Remove the abstract tag

Reply:
It's gone